backend/apps/volontulo/views/api.py

```diff
@@ -323,3 +323,23 @@ def post(self, request):  # pylint:disable=no-self-use,unused-argument
             send_copy_to_admin=False,
         )
         return Response({}, status.HTTP_201_CREATED)
+
+
+class PasswordChangeView(APIView):
+    """Password change view."""
+    permission_classes = (IsAuthenticated,)
+    authentication_classes = (CsrfExemptSessionAuthentication,)
+
+    def post(self, request):  # pylint:disable=no-self-use,unused-argument
```

Comment:
use `staticmethod` and omit `self` at all - use `request` directly insead of `self.reqeuest`